packages/delivery-utils/src/array-from-node-list.test.ts

```diff
@@ -0,0 +1,38 @@
+import { arrayFromNodeList } from './array-from-node-list';
+
+describe('Array From NodeList Test', () => {
+    document.body.innerHTML = `
+    <ul>
+        <li>Test 0</li>
+        <li>Test 1</li>
+        <li>Test 2</li>
+        <li>Test 3</li>
+    </ul>
+    `;
+
+    test('Given an empty NodeList, return an empty array', () => {
+        const converted = arrayFromNodeList(document.querySelectorAll('div'));
+
+        expect(converted.length).toBe(0);
+    });
+
+    test('Array method forEach() should exist on prototype of array', () => {
+        const converted = arrayFromNodeList(document.querySelectorAll('li'));
+
+        converted.forEach((convertedNode, index) => {
+            expect(convertedNode.innerHTML).toBe(`Test ${index}`);
+        });
```

Comment:
This structure of a test has some hazards. For example, if `forEach()` ended up being a noop function, no assertions would get evaluated, none would fail, and the test would pass by default.

You can mitigate this by either:
(1) using `expect.assertions(num)` to verify that exactly `num` assertions were checked, or
(2) testing `converted` against an array of expected values. Not being DRY is acceptable and even preferred in unit tests
```typescript
expect(converted).toHaveLength(4); // Tests if array

const values = converted.map((node) => node.innerHTML);
expect(values).toEqual(['Test 0', 'Test 1', 'Test 2', 'Test 3']);
```